src/imio/news/core/subscribers.py

```
@@ -98,14 +98,14 @@ def removed_newsfolder(obj, event):
     entity = get_entity_for_obj(obj)
     reload_faceted_config(entity, request)
     # @search endpoint invaldation
-    # invalidate_endpoint_search_cache(obj)
+    invalidate_endpoint_search_cache(obj)
 
 
 def added_news_item(obj, event):
```

Ruff (0.15.2)

[warning] Unused function argument: `event`

(ARG001)